src/community/ogcapi/ogcapi-features/src/main/java/org/geoserver/ogcapi/v1/features/FeatureConformance.java

```diff
@@ -0,0 +1,491 @@
+package org.geoserver.ogcapi.v1.features;
+
+import static org.geoserver.ogcapi.APIConformance.Level.DRAFT_STANDARD;
+import static org.geoserver.ogcapi.APIConformance.Level.STANDARD;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.geoserver.ogcapi.APIConformance;
+import org.geoserver.ogcapi.ConformanceClass;
+import org.geoserver.ogcapi.ConformanceInfo;
+import org.geoserver.wfs.WFSInfo;
+
+/** FeatureService configuration. */
+public class FeatureConformance extends ConformanceInfo<WFSInfo> {
+    public static String METADATA_KEY = "ogcapiFeatures";
+
+    public static final APIConformance CORE =
+            new APIConformance(
+                    "http://www.opengis.net/spec/ogcapi-features-1/1.0/conf/core", STANDARD);
+
+    // required resource formats
+    public static final APIConformance HTML =
+            CORE.extend("http://www.opengis.net/spec/ogcapi-features-1/1.0/conf/html");
+    public static final APIConformance GEOJSON =
+            CORE.extend("http://www.opengis.net/spec/ogcapi-features-1/1.0/conf/geojson");
+    public static final APIConformance OAS30 =
+            CORE.extend("http://www.opengis.net/spec/ogcapi-features-1/1.0/conf/oas30");
+
+    // optional output format from WFS
+    public static final APIConformance GML321 =
+            new APIConformance("http://schemas.opengis.net/gml/3.2.1/gml.xsd", STANDARD);
+
+    // not-implemented resource formats
+    public static final APIConformance GMLSF0 =
+            CORE.extend("http://www.opengis.net/spec/ogcapi-features-1/1.0/conf/gmlsf0");
+    public static final APIConformance GMLSF2 =
+            CORE.extend("http://www.opengis.net/spec/ogcapi-features-1/1.0/conf/gmlsf2");
+
+    // optional
+    public static final APIConformance CRS_BY_REFERENCE =
+            CORE.extend("http://www.opengis.net/spec/ogcapi-features-2/1.0/conf/crs");
+    public static final APIConformance FEATURES_FILTER =
+            CORE.extend(ConformanceClass.FEATURES_FILTER);
+    public static final APIConformance FILTER = CORE.extend(ConformanceClass.FILTER);
+    public static final APIConformance QUERYABLES = CORE.extend(ConformanceClass.QUERYABLES);
+
+    // draft
+    public static final APIConformance IDS =
+            new APIConformance(ConformanceClass.IDS, DRAFT_STANDARD);
+    public static final APIConformance SEARCH =
```

Comment:
So... when this one is not enabled, the search method in FEatureService should throw a 404?

Reply:
Done